docs/GOVERNANCE.md

```diff
@@ -159,53 +165,85 @@ The specifications produced will adhere to the following:
-- Create a PR that modifies the [README](README.md), and all specifications (ie.
-  \*.md files) that include a version string, to the new release version string.
-  Make sure to remove `-wip` from all of the version strings.
+
+- Determine the new release version string. It should be of the form:
+  `<subject>/vX.Y.Z`, e.g. `cloudevents/v1.0.4` or `subscriptions/v1.0.0`.
+
+- Create a PR (for the "main" branch") that:
+  - Modifies the repo's files to use the new version string appended with
+    `-rc#` as appropriate. Make sure to remove all `-wip` suffixes as needed.
+  - Merge the PR.
+  - Initiate a final review/test of the release.
+
+- Create A "release candidate" with the new release version string but with a
+  suffix of `-rc#` (release candidate). This will indicate that the authors
+  believe it is ready for final review/testing. This will be true for updates
+  to existing specifications and for new specifications.
+
+- When review/testing is completed, create a PR (for the "main" branch") that:
+  - Modifies the repo's files to use the new version string (without `-rc#`)
+    as appropriate..
+  - Update [RELEASES.md](RELEASES.md) to mention the new release, and
+    reference the yet-to-be-created release tag.
+  - Update the appropriate `*/RELEASE_NOTES.md` file with the changes
+    for the release. The list can be generated via:
+    `git log --pretty=format:%s main...cloudevents/v1.0.3 | grep -v "Merge pull"`
+    by replacing "cloudevents/v1.0.3" with the name of the previous release.
+    Or, use GitHub's
+    [new release](https://github.com/cloudevents/spec/releases/new) process
+    to generate the list without actually creating the release yet.
+
 - Merge the PR.
-- Create a [new release](https://github.com/cloudevents/spec/releases/new):
-  - Choose a "Tag version" of the form: `<subject>/vX.Y.Z`, e.g.
-    `cloudevents/v1.0.4` or `subscriptions/v1.0.0`
-  - Target should be `main`, the default value
-  - Release title should be the same as the Tag - `<subject>/vX.Y.Z`
-  - Add some descriptive text, or the list of PRs that have been merged since
-    the previous release. The git query to get the list commits since the last
-    release is:
-    `git log --pretty=format:%s main...v0.1 | grep -v "Merge pull"`.
-    Just replace "v0.1" with the name of the previous release.
-  - Press `Publish release` button
-- Create an "announcement" highlighting the key features of the new release and
-  any potential noteworthy activities of the group:
-  - Send it to the mailing list
-  - Announce the release on our
-    [twitter account](http://twitter.com/cloudeventsio)
+  - Note that the link checker should fail since any references to the new
+    release tag will not be valid yet. This is expected.
+
+- Create a new branch with the same name as the new release version string
+  appended with `-branch` (e.g. `<subject>/vX.Y.Z-branch`).
+  - Use GitHub to create a
```

Comment:
RC Tag name:  <subject>@vX.Y.Z-rc1
Branch name:  <subject>@vX.Y.Z-branch
Tag name: <subject>@vX.Y.Z

Reply:
But with the subject still, right?

Reply:
Should this comment be included in the document somewhere? It might be useful just after "Each release will have both a tag and a branch".

Reply:
yup - just added